rfcs/react-components/convergence/static-classes-on-slots.md

```diff
@@ -0,0 +1,92 @@
+# RFC: Conventions for static class names on slots
+
+Contributors: @behowell
+
+History:
+
+- [PR #21206](https://github.com/microsoft/fluentui/pull/21206) - Initial submission
+
+## Summary
+
+This RFC proposes adding a static className to every slot, using the naming format `"fui-{ComponentName}__{slotName}"`.
+
+## Problem statement
+
+It can be useful to have a static class name on the slot of a component, including by users of our library if they want to write static styles targeting a slot.
```

Comment:
```suggestion
It can be useful to have a static class name on the slot of a component, including by users of our library if they want to write static styles targeting a slot, or to track down the rendered slot while debugging.
```